### PR TITLE
prevent trigger in dragging

### PR DESCRIPTION
**Pull request by janryWang**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "react-contextmenu",
-  "version": "2.0.0-alpha.2",
+  "version": "2.0.0-alpha.3",
```
Comment (vkbansal) [inline package.json]: revert this change please

**Comment by janryWang** [comment]
done

File: src/ContextMenuTrigger.js [hunk 2]
```diff
@@ -49,26 +53,37 @@ export default class ContextMenuTrigger extends Component {
...
 
+
     handleTouchEnd = (event) => {
         event.preventDefault();
         clearTimeout(this.touchstartTimeoutId)
     }
 
+    handleMouseMove = (event)=>{
+        event.persist()
+        this.mouseMoveEvent = event
```
Comment (vkbansal) [inline src/ContextMenuTrigger.js]: events in React are synthetic events and are reused.
I believe this will not work as expected.

why not use this instead?
```js
 this.mousePosition = {
    x:event.pageX,
    y:event.pageY
}
```

Comment (janryWang) [inline src/ContextMenuTrigger.js]: You do not need to doubt, it certainly worked, in fact, I don‘t want to cache the coordinate values, I hope to get the real-time coordinates.

**Comment by vkbansal** [comment]
Why close this?

**Comment by janryWang** [comment]
Sorry, Because you reply to me too slow, I don't have time to spend with wait for your reply, now, I am already copy your code to my project.

**Comment by vkbansal** [comment]
For your kind information, I have a full-time job, and a personal life and work on this project in my **free time**. So Sorry, for late replies (and having a full time job).

I guess you are new to open source. Anyways thanks for your contribution

**Comment by janryWang** [comment]
Okay,Thanks
